core/src/main/java/jenkins/management/AdministrativeMonitorsConfiguration.java

```diff
@@ -43,7 +43,12 @@
     public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
         for (AdministrativeMonitor am : AdministrativeMonitor.all()) {
             try {
-                boolean disable = !json.getJSONArray("administrativeMonitor").contains(am.id);
+                boolean disable;
+                if(json.optJSONArray("administrativeMonitor") != null) {
```

Comment:
I would rather retrieve the array once. The `optString` call may also return null and cause NPE in th second clause. So I've inverted the expression in my sample

```
var monitors = optJSONArray("administrativeMonitor")
if (monitors != null) {
  disable = monitors.contains(...)
} else {
  disable = am.id.equals(json.optString("administrativeMonitor"))
}
```